src/manage.py

```diff
@@ -11,7 +11,7 @@
         # issue is really that Django is missing to avoid masking other
         # exceptions on Python 2.
         try:
-            import django
```

Comment:
Gezien de comment hierboven lijkt deze try/except een compatibility-fix voor python 2.7. Misschien kunnen we hem inmiddels wel helemaal verwijderen.